boa_tester/src/main.rs

```diff
@@ -375,6 +397,9 @@ struct Test {
     name: Box<str>,
     description: Box<str>,
     esid: Option<Box<str>>,
+    spec_version: Option<i8>,
+    es5id: Option<Box<str>>,
+    es6id: Option<Box<str>>,
```

Comment:
We can remove those fields, we have enough information to filter by version with `spec_version`